documentation/examples/prometheus-kubernetes.yml

```diff
@@ -202,6 +202,34 @@ scrape_configs:
   - source_labels: [__meta_kubernetes_service_name]
     target_label: kubernetes_name
 
+- job_name: 'kubernetes-ingresses'
+
+  metrics_path: /probe
+  params:
+    module: [http_2xx]
+
+  kubernetes_sd_configs:
+    - role: ingress
+
+  relabel_configs:
+    - source_labels: [__meta_kubernetes_ingress_annotation_prometheus_io_probe]
+      action: keep
+      regex: true
+    - source_labels: [__meta_kubernetes_ingress_scheme,__address__,__meta_kubernetes_ingress_path]
+      regex: (.+);(.+);(.+)
+      replacement: ${1}://${2}${3}
+      target_label: __param_target
+    - target_label: __address__
+      replacement: blackbox
```

Comment:
What about a clearer example that this is intended to be the address of a blackbox exporter? Like `blackbox-exporter.company.org:9115` or so?

Reply:
There is already a blackbox-exporter example right above. But agreed, with something like that it's easier to understand. Changing it.